From 881f134172e60f2b2bc4ac4ebabaf2120e46ad86 Mon Sep 17 00:00:00 2001
From: arokem <arokem@gmail.com>
Date: Mon, 12 Oct 2015 19:27:19 -0700
Subject: [PATCH 2/2] BF: For numpy 1.6, we might need to cast this as float.

---
 dipy/core/sphere.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dipy/core/sphere.py b/dipy/core/sphere.py
index 51d4bc2..675db12 100644
--- a/dipy/core/sphere.py
+++ b/dipy/core/sphere.py
@@ -401,7 +401,7 @@ def _switch_vertex(index1, index2, vertices):
     A = vertices[index1]
     B = vertices[index2]
     is_far = (A * B).sum(-1) < 0
-    index2[is_far] = index2[is_far] + (n / 2)
+    index2[is_far] = index2[is_far] + (n / 2.0)
     index2 %= n
 
 
-- 
2.6.2

